### PR TITLE
Add ratings information and number of users/downloads to the add-on search/detail APIs

### LINKED ISSUE #3021: Return ratings and downloads/users information in the search/detail API
Search and detail API need to return the number of ratings, the average rating, the number of users and the number of weekly downloads for each add-on.

Supports mozilla/addons/issues/151

### PR DESCRIPTION
**Pull request by diox**
Fix #3021

### CODE REVIEW

File: docs/topics/api/addons.rst [hunk 1]
```diff
@@ -90,19 +91,22 @@ This endpoint allows you to fetch a specific add-on by id, slug or guid.
     :>json string current_version.files[].url: The (absolute) URL to download a file.
     :>json string default_locale: The add-on default locale for translations.
     :>json string|object|null description: The add-on description.
-    :>json string edit_url: The URL to the developer edit page for this add-on.
+    :>json string edit_url: The URL to the developer edit page for the add-on.
     :>json string guid: The add-on `extension identifier <https://developer.mozilla.org/en-US/Add-ons/Install_Manifests#id>`_.
     :>json string|object|null homepage: The add-on homepage.
-    :>json string icon_url: The URL to icon for this add-on (cache-busting query string included).
+    :>json string icon_url: The URL to icon for the add-on (including a cachebusting query string).
```
Comment (EnTeQuAk) [inline docs/topics/api/addons.rst]: I think "cache-busting" is more correct but don't quote me on that.

**Comment by EnTeQuAk** [comment]
Not sure if this was ready to take for review but from what I see: r+wc - needs rebasing though.

File: docs/topics/api/addons.rst (hunk 1 above)
Comment (diox) [inline docs/topics/api/addons.rst]: I googled the two and got more results with "cachebusting", so I went with that :)

**Comment by diox** [comment]
It was ready, thanks :) rebased & merged.

**Pull request merged**